README.md

```diff
@@ -62,7 +62,9 @@ Every argument is optional.
 | [remove-stale-when-updated](#remove-stale-when-updated)             | Remove stale label from issues/PRs on updates                               | `true`                |
 | [remove-issue-stale-when-updated](#remove-issue-stale-when-updated) | Remove stale label from issues on updates/comments                          |                       |
 | [remove-pr-stale-when-updated](#remove-pr-stale-when-updated)       | Remove stale label from PRs on updates/comments                             |                       |
+| [labels-to-add-when-stale](#labels-to-add-when-stale)           | Add specified labels from issues/PRs when they become stale               |                       |
```

Comment:
We already have similar inputs (`stale-issue-label` and `stale-pr-label`). Maybe we should update these inputs instead of introducing a new one? BTW were these changes requested? I thought we need to add only an option to remove